djstripe/models.py

```diff
@@ -45,12 +45,41 @@ def convert_tstamp(response, field_name=None):
 
 
 class StripeObject(TimeStampedModel):
+    # This must be defined in descendants of this model/mixin
+    # e.g. "Event", "Charge", "Customer", etc.
+    stripe_api_name = None
 
     stripe_id = models.CharField(max_length=50, unique=True)
 
-    class Meta:
+    class Meta(object):
```

Comment:
Sounds good. There are only three Meta classes in the package, so I'll just add the updates as my next commit to the API upgrade branch. Better to future proof now.